### DISCUSSION TITLE
Allowing for adding banners on Nova

### DISCUSSION THREAD

**Discussion by FoksVHox** [laravel/nova-issues#5803] [Ideas & Feature Requests]
Greetings!

It would certainly be a welcome addition to Nova if we could incorporate banners similar to those in Jetstream, albeit with an added layer of functionality.

The concept would mirror the banners seen in Jetstream, offering both persistent banners and those that reset upon entering new routes.

**Reasoning**
This proposed feature holds significant value for applications wherein external services might impact the application's functionality. Moreover, it could prove invaluable in scenarios where issues are being addressed and services are temporarily down for maintenance.

**Comment by davidhemphill** [comment]
What would this offer over the existing Notification Center inside Nova?

**Reply by FoksVHox** [reply]
@davidhemphill This would offer much of the same functionality, however, with the Notification Center, the important notifications, could easily be overseen when they're with all the other notification.

This feature could be an extension of the Notification Center.
